Userland/Libraries/LibWeb/HTML/HTMLHyperlinkElementUtils.cpp

```diff
@@ -114,7 +114,7 @@ void HTMLHyperlinkElementUtils::set_username(DeprecatedString username)
         return;
 
     // 4. Set the username given this’s URL and the given value.
-    url->set_username(AK::URL::percent_encode(username, AK::URL::PercentEncodeSet::Userinfo));
+    url->set_username(username);
```

Comment:
The commit message here says "URL::set_password will apply a percent encode" but the commit is only updating `set_username`. Is there some behind the scenes trickery, or is that a typo?

Reply:
Right, that's a typo, thanks!